spec/unit/deployment/config/loader_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+require 'r10k/deployment/config/loader'
+
+describe R10K::Deployment::Config::Loader do
+
+  it "creates R10K::Deployment::Config::Loader instances" do
+    subject = described_class.new()
+    expect(subject).to be_a_kind_of R10K::Deployment::Config::Loader 
+  end
+
+  it "includes /etc/puppetlabs/r10k/r10k.yaml in the loadpath" do
+    subject = described_class.new()
```

Comment:
RSpec automatically creates a `subject` helper by calling `described_class.new`; we don't need to explicitly set subject to use it.

For reference, you can also set an explicit subject:

```ruby
  subject do
    described_class.new('something', 'with', specific', 'args')
```